conference/talk_voting.py

```diff
@@ -24,11 +23,22 @@ def talk_voting(request):
             }
         )
 
+    filter = request.GET.get('filter')
+    if filter == 'voted':
+        extra_filters = [Q(id__in=VotoTalk.objects.filter(user=request.user).values('talk_id'))]
+    elif filter == 'not-voted':
+        extra_filters = [~Q(id__in=VotoTalk.objects.filter(user=request.user).values('talk_id'))]
+    else:
+        filter = 'all'
+        extra_filters = []
+
     talks = (
         Talk.objects.filter(
             Q(conference=current_conference.code) & ~Q(created_by=request.user)
-        )
-        .order_by("?")
+            & Q(admin_type='') & Q(status=TALK_STATUS.proposed)
```

Comment:
not sure if this was blacked already, but I think one '& ...' per line would be more readable. WDYT?